tasks/eth/ink-001-permissionless-proofs/VALIDATION.md

```diff
@@ -0,0 +1,46 @@
+# Validation
```

Comment:
the `.env` is using the `SAFE_NONCE=6`
```
SAFE_NONCE=6
```
Then, we should expect some storage validation about the safe state transition in this file.

Reply:
Added this in my most recent commit